CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1012,7 +1104,11 @@ local function CreateSelector(selectionStringTable, startPosition)
 					this.Selections[this.CurrentIndex].TextTransparency = 0
 				else
 					if GuiService.SelectedCoreObject ~= nil and isAutoSelectButton[GuiService.SelectedCoreObject] then
-						GuiService.SelectedCoreObject = this.SelectorFrame
+						if UserInputService.VREnabled and fixSettingsMenuVR then
+							this.Selections[this.CurrentIndex].TextTransparency = 0
```

Comment:
I think it gets set somewhere else anyway, this just reproduces the functionality for VR since mouse hover events don't quite work right.

That reminds me, we should see if we can get those working some time and remove these special cases.